ls1intum/artemis-ios
# `General`: Release 1.2.0

## Fixed
- Many communication notifications opening with "This link is not supported" error
- Lecture/Exercise Post notifications having wrong content
- Lecture/Exercise Post Reply notifications having no content

## New
- Conversation Unread Indicator Redesign
- Ability to directly navigate to Communication Channel from Lecture/Exercise Details
- Ability to pin messages and resolve threads
- Swipe to Reply
- Quick Actions Bar for main Message in Thread
- Ability to view Reaction Authors via long press
- Usability improvements for Conversation Info

## Testing
### How to install
1. <a href="https://testflight.apple.com/join/canwoVCk">Join Artemis TestFlight Beta: testflight.apple.com/join/canwoVCk</a>
2. Download & Install Latest Build of Version 1.2

### What to test
- Sending messages
- Replying to messages
- Tapping on notifications
- Navigating from Exercise & Lecture Details to Channel
- View Reaction Authors (long press on a reaction)
- React to messages
- Verify that pinned messages as well as resolving replies are highlighted

You can use a test server by entering its url on the login screen under "Select university">"Custom instance".

### Known issues
- You cannot open conversations on iOS 18 Beta 6 or later as well as Beta 3 and older. Please test on iOS 17 or iOS 18 Beta 4-5.
